### PR TITLE
Pizza bytes 

### PR DESCRIPTION
**Pull request by kathinka**
## Netlify link

https://pizzabytez.netlify.app/
https://pizzabytez.netlify.app/switch

### CODE REVIEW

File: code/index.html
Comment (AntonellaMorittu) [inline code/index.html:1]: Please format your code to have the right spaces and indentation 👍

File: code/script.js [hunk 1]
```diff
@@ -1,19 +1,178 @@
...
+
+}
+
+
 
 // Step 5 - Order confirmation
 // Your code goes here
+
+function confirmOrder() {
+  let confirm = parseInt(prompt("You have ordered a " + age + " size " + food + ". \ncost: " + foodCost + " Bytecoin.\nPress 1 if this is correct. \nPress 2 if it's wrong."));
```
Comment (AntonellaMorittu) [inline code/script.js:162]: this is correct, but have you thought about using template literals here to improve readability? :)

File: code/switch.js
Comment (AntonellaMorittu) [inline code/switch.js:1]: Please format your code to have the right spaces and indentation 👍

File: code/switch.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    else if (age >= 16) {
+        order.push(orderType[0].cost);
+        alert("Seems like you are old enough for a regular portion.\n The cost is " + order[1] + " ByteCoin");
+        checkOut();
+    }
+    else {
+        alert("invalid");
+        ageCheck();
+    }
+}
```
Comment (AntonellaMorittu) [inline code/switch.js:93]: this checks are very good, would have been nice to see it in action on your deployed version! 👍

File: code/switch.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                order.push(menu[8].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            default:
+                alert("Sorry that's not on the  menu")
+                break;
+        }
+        break;
+}
```
Comment (AntonellaMorittu) [inline code/switch.js:224]: Make sure to use a formatter to have the correct spacing and indentation, and change the tab size settings from 4 to 2😄 you can see now there is a lot of empty spaces in the code that compromise the readability ✨

**Review by AntonellaMorittu — APPROVED** [review]
Hi Kathinka, good job this week 🚀 the code meets the requirements and stretch goals, that's great! I would have liked to see the switch.js script executed in your deployed version. I left some comments specifically on some parts, please take a look at them :) Thank you ⭐

File: code/script.js (hunk 1 above)
Comment (kathinka) [reply code/script.js:162]: good suggestion

File: code/switch.js (hunk 2 above)
Comment (kathinka) [reply code/switch.js:93]: hmm.. did you not see the provided links?
https://pizzabytez.netlify.app/
https://pizzabytez.netlify.app/switch

also after completion, you should see a button who will take you to the other example.

Comment (kathinka) [reply code/switch.js:93]: switched the scripts so now the switch versjon loads first.

**Pull request closed without merging**